ush/config.sh.RTMA_CONUS_3km

```diff
@@ -260,5 +260,5 @@ if [[ "$MACHINE" == "wcoss_dell_p3" ]]; then
 fi
 # Wait longer for rrfsdas
 if [[ "$MACHINE" == "jet" ]] && [[ $regional_ensemble_option -eq 5 ]] ; then
-  START_TIME_ANAL="5:00:00"
+#  START_TIME_ANAL="5:00:00"
 fi
```

Comment:
I believe this entire if block will need to be commented out to prevent syntax errors. Alternatively, if this variable is still needed, it could be added to config_defaults.